Correct internal gesture ID for Seika Notetaker (#17047)
nvaccess/nvda: user_docs/en/changes.md

Reviewer: Please add an entry to Changes for developers

Reviewer: @school510587 you need to merge master into your branch, then add an entry to the "Bug Fixes" section describing the user-facing changes, and an entry to the "Changes for Developers" section describing what you have changed internally.

* Improvements when editing in Microsoft PowerPoint:
  * Caret reporting no longer breaks when text contains wide characters, such as emoji. (#17006 , @LeonarddeR)
  * Character location reporting is now accurate (e.g. when pressing `NVDA+Delete`. (#9941, @LeonarddeR)

> * Correct IDs of single space and space with dots gestures for Seika Notetaker on the input gestures dialog. (#17047, @school510587)

Reviewer: ```suggestion
* When using the Seika Notetaker, space and space with dots gestures are now displayed correctly in the Input Gestures dialog. (#17047, @school510587)
```

  * It is now possible to use braille display routing keys to move the text cursor. (#9101)
  * It is now possible to use the review cursor selection commands to select text.
* Updating NVDA while add-on updates are pending no longer results in the add-on being removed. (#16837)

> * Correct internal gesture ID for Seika Notetaker. (#17047, @school510587)

Reviewer: This needs to be moved to 2025.1. It also needs to explain the user-facing change.

  * A `toggleBooleanValue` helper function has been added to `globalCommands`.
  It can be used in scripts to report the result when a boolean is toggled in `config.conf`
* Removed the requirement to indent function parameter lists by two tabs from NVDA's Coding Standards, to be compatible with modern automatic linting. (#17126, XLTechie)
* Correct the internal IDs of the braille input gesture objects for Seika Notetaker. (#17047, @school510587)
  * Parameter space of class InputGesture constructor becomes an integer between 0 and 3.

>   * Gestures involving br(seikantk):BACKSPACE and/or br(seikantk):SPACE can be correctly linked to command scripts.

Reviewer 2: Sorry, this should be under API breaking changes.
The issue is that you are breaking the API, we don't need a general description of the code changes here.
Please read [this doc](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/deprecations.md) for more information.

The message should be something like

```py
* The `space` keyword argument for `brailleDisplayDrivers.seikantk.BrailleDisplayDriver` now expects an `int` not a `bool`. (#17047)
```
